apps/nextjs-website/src/messages/it.json

```
@@ -101,7 +101,10 @@
         "speakersTitle": "Ospiti",
         "questionList": {
             "createdAt": "Ora creazione",
-            "text": "Domanda"
+            "text": "Domanda",
+            "hiddenBy": "Nascosta da: ",
+            "hiddenByMe": "Hai nascosto questa domanda",
+            "highlightedBy": "Evidenziato da: "
```

```suggestion
            "text": "Domanda",
            "hiddenBy": "Nascosta da",
            "hiddenByMe": "Hai nascosto questa domanda",
            "highlightedBy": "Evidenziato da"
```
What do you think to move the white spaces and colons where you use the translation?
In this case you will have something like
```
<Typography>`${t('hiddenBy')}: ${name}`</Typography>
```